lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -0,0 +1,304 @@
+import BitcoinClient from '../BitcoinClient';
+import BitcoinError from '../BitcoinError';
+import BitcoinLockTransactionModel from '../models/BitcoinLockTransactionModel';
+import ErrorCode from '../ErrorCode';
+import LockIdentifier from '../models/LockIdentifierModel';
+import LockIdentifierSerializer from './LockIdentifierSerializer';
+import LockResolver from './LockResolver';
+import LockTransactionModel from './../models/LockTransactionModel';
+import LockTransactionType from './../enums/LockTransactionType';
+import MongoDbLockTransactionStore from './MongoDbLockTransactionStore';
+import ValueTimeLockModel from './../../common/models/ValueTimeLockModel';
+
+/**
+ * Structure (internal to this class) to track the information about lock.
+ */
+interface LockInformation {

```

Comment:
Nit: For consistency, we have been using the abbreviated form.
```suggestion
interface LockInfo {

```